pykg2tbl/j2/templates/skos-collection-member-count.sparql

```diff
@@ -0,0 +1,13 @@
+{# Jinja Template --> Sparql Query
+ | Count the number of members in a skos collection
+#}
```

Comment:
please apply the same inline documentation pattern for the variables in use 
(see the other template examples)